.editorconfig

```diff
@@ -0,0 +1,36 @@
+# http://editorconfig.org
```

Comment:
I suggest to use [Javascript](https://standardjs.com/) or [Airbnb](https://github.com/airbnb/javascript) styleguide without any custom changes because it is two major styleguides in the ecosystem. Is it possible to check and refactor this file and the following files?

Please make sure we are using one of these two styleguides without custom changes and improvements. The choice is on your own I am ok with both.